plugins/pubIds/urn/URNPubIdPlugin.inc.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+/**
+ * @file plugins/pubIds/urn/URNPubIdPlugin.inc.php
+ *
+ * Copyright (c) 2014-2016 Simon Fraser University Library
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class URNPubIdPlugin
+ * @ingroup plugins_pubIds_urn
+ *
+ * @brief URN plugin class
+ */
+
+
+import('classes.plugins.PubIdPlugin');
+
+class URNPubIdPlugin extends PubIdPlugin {
+
+	//
+	// Implement template methods from Plugin.
+	//
+	/**
+	 * @copydoc Plugin::getDisplayName()
+	 */
+	function getDisplayName() {
+		return __('plugins.pubIds.urn.displayName');
+	}
+
+	/**
+	 * @copydoc Plugin::getDescription()
+	 */
+	function getDescription() {
+		return __('plugins.pubIds.urn.description');
+	}
+
+	/**
+	 * @copydoc Plugin::getTemplatePath()
+	 * @param $inCore boolean True iff a core template should be preferred
```

Comment:
When you use `copydoc`, parameter descriptions get inherited from the parent class -- no need to describe this here.